runtime/pack/dist/opt/cfilter/plugin/cfilter.lua

```diff
@@ -48,7 +49,7 @@ Qf_filter = function(qf, searchpat, bang)
     Xsetlist = function(...)
       return vim.fn['setqflist'](...)
     end
-    cmd = ':Cfilter' .. bang
+    cmd = string.format([=[:Cfilter{bang}]=])
```

Comment:
Since these files are generated, I'd prefer not to make any code changes here. 

Simply ignore the whole file.

Reply:
OK, I'll just keep the disabling parts.